src/backends/beamcoder.ts

```diff
@@ -210,11 +217,8 @@ export class BeamcoderExtractor extends BaseExtractor implements Extractor {
      */
```

Comment:
Maybe add something along these lines?

```suggestion
     * This is the duration of the first video stream in the file.
     */
```